.gitignore

```diff
@@ -4,11 +4,13 @@
 ga4gh-example-data*
 ga4gh-downloaded-data*
 scripts/auth.yml
+ga4gh/_version.py
```

Comment:
Are these changes incidental to sequence annotation? Please make a PR against master if so.